examples/get_efd_data.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
There should be a 1-2 sentence description of what this notebook does at the top of the notebook.